plugins/Phaser/PhaserEffect.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * PhaserEffect.h
+ *
+ * Copyright (c) 2019 Lost Robot <r94231@gmail.com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+
+#ifndef PHASER_H
+#define PHASER_H
+
+#include "Effect.h"
+#include "PhaserControls.h"
+#include "ValueBuffer.h"
+#include "../Flanger/QuadratureLfo.h"
+#include "RmsHelper.h"
+
+class PhaserEffect : public Effect
+{
+	Q_OBJECT
+public:
+	PhaserEffect(Model * parent, const Descriptor::SubPluginFeatures::Key * key);
+	virtual ~PhaserEffect();
+	virtual bool processAudioBuffer(sampleFrame * buf, const fpp_t frames);
+
+	virtual EffectControls* controls()
+	{
+		return &m_phaserControls;
+	}
+
+	float m_realCutoff[2] = {0, 0};
```

Comment:
Rather than having `m_realCutoff` be public, it would better to keep it private and expose it through a const getter function.